tests/unit/code_test.py

```diff
@@ -19,7 +19,7 @@ def test_train_model():
     run = Mock(Run)
     reg = train_model(run, data, alpha=1.2)
 
-    run.log.assert_called_with("mse", 0.029843893480256872,
+    run.log.assert_called_with("mse", 0.029843893480257067,
```

Comment:
This change causing unit test to fail

Reply:
Seems that a minor update in sklearn is causing this difference. The issue is that unit tests are currently running with mlopspython from master, but after merge mlopspython will be rebuilt from this conda env...